src/collections/_documentation/guides/manage-event-stream/index.md

```diff
@@ -133,10 +133,10 @@ Applying the proper filters, SDK configuration, and rate limits is an iterative
 
 ### > **How can I see a breakdown of incoming events?**
 
-Opening the `Stats` view from the left side navigation bar displays details about the total number of events Sentry has received across your entire organization.  The numerical weekly report breaks down the events by project into three categories:
+Opening the `Stats` view from the left side navigation bar displays details about the total number of events Sentry has received across your entire organization. The weekly numerical report breaks down the events by project into three categories:
```

Comment:
Is it numerical by week or numerical by report? "numerical" goes in front of the specific thing being numbered.

Reply:
good point. let me rework this one.